CONTRIBUTING.md

```diff
@@ -1,24 +1,69 @@
-Dune is an community orientated open source project. It was originally
-developed at [Jane Street][js] and is now maintained by Jane Street,
-[Tarides][tarides] as well as several developers from the OCaml
-community.
+Dune is an community orientated open source project. It was originally developed
+at [Jane Street][js] and is now maintained by Jane Street and [Tarides][tarides]
+together with several developers from the OCaml community.
 
-Contributions to Dune are welcome and should be submitted via GitHub
-pull requests against the `main` branch. See [./doc/hacking.rst][hack]
-for a guide to getting started on the code base.
+Filing issues
+=============
 
-Dune is distributed under the MIT license and contributors are
-required to sign their work in order to certify that they have the
-right to submit it under this license. See the following section for
-more details.
+The easiest way to contribute is to share your input on [issues][issues].
+Feedback on bug reports, feature requests, and documentation is welcome and
+appreciated. If you don't find a preexisting issue discussing your topic, then
+please [file a new issue][file an issue].
+
+[file an issue]: https://github.com/ocaml/dune/issues/new/choose
```

Comment:
We should also mention discussions.

Reply:
What would we like to ask people to use discuss for?